holesky/README.md

```diff
@@ -219,3 +219,16 @@ Make sure your `.env` file still has correct values in the [TODO](https://github
 ```
 docker compose up -d
 ```
+
+## Running your node without giving access to operator ECDSA keys
+In [EigenDA v0.6.1](https://github.com/Layr-Labs/eigenda-operator-setup/releases/tag/v0.6.1), we added a feature where you can configure your node so that it doesn't need operator's ECDSA keys to run.
+Your node still need access to BLS keys for attestation purposes.
+>**_NOTE:_** You still need ECDSA and BLS keys to opt-in to EigenDA.
+
+To enable this feature by using our setup, follow the below commands:
+* Remove the `"${NODE_ECDSA_KEY_FILE_HOST}:/app/operator_keys/ecdsa_key.json:readonly"` mount from `docker-compose.yml` file.
```

Comment:
Could we avoid the the instruction to modify docker-compose? Or is it necessary? Asking only b/c if we can avoid users modifying docker-compose that makes our life simpler in terms of user support. If we leave the line - the only damage will be that the non-existent file is mounted as a directory?